docs/src/app/components/LeftNav/ExampleOpenRight.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import LeftNav from 'material-ui/lib/left-nav';
+import AppBar from 'material-ui/lib/app-bar';
+import RaisedButton from 'material-ui/lib/raised-button';
+
+export default class LeftNavOpenRightExample extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {open: false};
+  }
+
+  handleToggle = () => this.setState({open: !this.state.open});
+
+  render() {
+    return (
+      <div>
+        <RaisedButton
+          label="Controlled LeftNav That Opens From Right"
+          onTouchTap={this.handleToggle} />
+        <LeftNav width={450} openRight={true} open={this.state.open} >
```

Comment:
We would need to make this number unitless. It could be `px` or `%`.
That's require some work. Same, we would have to use `px` or `%` for the transform3d property.